pkg/repositories/gormimpl/execution_repo.go

```diff
@@ -88,10 +90,21 @@ func (r *ExecutionRepo) List(ctx context.Context, input interfaces.ListResourceI
 			taskTableName, executionTableName, taskTableName))
 	}
 
-	// Apply filters
-	tx, err := applyScopedFilters(tx, input.InlineFilters, input.MapFilters)
-	if err != nil {
-		return interfaces.ExecutionCollectionOutput{}, err
+	var stateFilterExists bool
```

Comment:
since this is business logic, can we move the state filter injection to the manager layer?

Reply:
Done